### PR TITLE
Edn/dv: docs, integrate csrng_agent

### PR DESCRIPTION
**Pull request by senelson7**

### CODE REVIEW

File: hw/ip/csrng/data/csrng.prj.hjson [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
     name:               "csrng",
     design_spec:        "hw/ip/csrng/doc",
-    dv_doc:            "hw/ip/csrng/doc/dv",
+    dv_doc:             "hw/ip/csrng/doc/dv",
     hw_checklist:       "hw/ip/csrng/doc/checklist",
     version:            "0.5",
     life_stage:         "L1",
     design_stage:       "D1",
-    verification_stage: "V0",
+    verification_stage: "V1",
```
Comment (weicaiyang) [inline hw/ip/csrng/data/csrng.prj.hjson:13]: perhaps move this update to a separate PR. And this one usually goes with some update in checklist.md

**Review by weicaiyang — COMMENTED** [review]
code changes LGTM
can you split those CSRNG update to a separate PR and also remove those "Merge branch" commits.
BTW, please also add some brief description about the change in the PR, which will be helpful to understand this PR. Thanks

**Comment by senelson7** [comment]
This PR now just contains the edn/dv update commit. Should have been that way before, probably inadvertently did something wrong. Anyways, this PR just updates the csrng_agent to remove the 3rd push_pull_agent that we determined it doesn't need and integrates it properly into the edn env/tb. I'll do a separate PR for the csrng v0->v1 stuff.

File: hw/ip/edn/data/edn_testplan.hjson [hunk 2]
```diff
@@ -24,10 +23,49 @@
...
+            Verify boot-time request mode behaves as predicted.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: genbits
+      desc: '''
+            Verify genbits input is transferred to endpoint as predicted.
+            Verify fips bit(s) are properly transferred to endpoint.
```
Comment (mwbranstad) [inline hw/ip/edn/data/edn_testplan.hjson:50]: Verify that multiple endpoints can request for entropy, and in random order of requesting.

**Review by mwbranstad — APPROVED** [review]
One test plan item added, otherwise LGTM.

**Review by weicaiyang — APPROVED** [review]
LGTM
